include/parser/string-parser.hpp

```diff
@@ -406,10 +375,8 @@ class StringParserEngine {
   }
 
   // This methods decodes a C-like escape character.
-  static bool parseEscapeCharacter(const String& inputString,
-                                   size_t& index,
-                                   Output& output,
-                                   CompileState& state) {
+  static bool parseEscapeCharacter(const String& inputString, size_t& index,
+                                   Output& output, CompileState& state) {
```

Comment:
I see a C-style cast below, my eyes are bleeding :( In fact the cast is not necessary at all :)